ui/components/app/deposit-popover/on-ramp-item.js

```diff
@@ -17,35 +23,49 @@ const OnRampItem = ({
     return null;
   }
   return (
-    <Box paddingRight={6} paddingLeft={6}>
+    <Box className="on-ramp-item" as="li" marginRight={6} marginLeft={6}>
       <Box
         paddingTop={6}
         paddingBottom={6}
-        style={{
-          borderBottomSize: '1px',
-          borderBottomColor: COLORS.BORDER_MUTED,
-        }}
+        display={DISPLAY.FLEX}
+        alignItems={ALIGN_ITEMS.CENTER}
+        flexWrap={FLEX_WRAP.WRAP}
       >
-        <Box width={FRACTIONS.HALF}>{logo}</Box>
-        <Typography
-          variant="h6"
-          fontWeight="bold"
-          boxProps={{
-            paddingTop: 2,
-            paddingBottom: 2,
-          }}
+        <Box
+          width={[
+            BLOCK_SIZES.ONE_THIRD,
+            BLOCK_SIZES.ONE_THIRD,
+            BLOCK_SIZES.ONE_FIFTH,
+          ]}
```

Comment:
This change is to fix the size of the logo since with one third as a size, the logo is looking too small on sm and md screens.

```suggestion
          width={[BLOCK_SIZES.HALF, BLOCK_SIZES.HALF, BLOCK_SIZES.ONE_FIFTH]}
```